CMakeLists.txt

```diff
@@ -44,44 +44,50 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
-file(GLOB SOURCES "edlib/src/*.cpp")
+set(EDLIB_SOURCE_DIR edlib/src/)
```

Comment:
I am not sure about the best approach. I just commented some parts of it temporarily to be able to run it, I mean the parts which were related to building and installing the library. As far as I know, since we are using templates it is not possible to make a library in that way. I think it is better to ask the other contributor.